core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -105,6 +105,10 @@
                                   style="margin-left: 10px;">
         </exploration-summary-tile>
       </div>
+
+      <div>
+        <a ng-href="/collection/{{collectionId}}" class="conversation-skin-back-to-collection" translate="I18N_PLAYER_BACK_TO_COLLECTION"></a>
```

Comment:
I thought that this would work, because $scope.collectionId should exist, and angular would render it in. However, while testing, this doesn't seem to work, and I am not sure what is wrong.